browser/js/experiences/experienceDetail.controller.js

```diff
@@ -14,11 +15,39 @@ app.controller('experienceDetailCTRL', function ($scope, experiencesFactory, exp
 					content: '<h2>Item added to Cart</h2>'
 				});//end ngToast.create
 			});
-
 	};
 
 $scope.experienceInStock = function () {
 	return $scope.tempQuantity > 0;
 };
 
-});
+$scope.rate = experience.ratingAverage;
+ $scope.max = 5;
+ $scope.isReadonly = true;
+
+
+ $scope.ratingStates = [
+	 {stateOn: 'glyphicon-ok-sign', stateOff: 'glyphicon-ok-circle'},
+	 {stateOn: 'glyphicon-star', stateOff: 'glyphicon-star-empty'},
+	 {stateOn: 'glyphicon-heart', stateOff: 'glyphicon-ban-circle'},
+	 {stateOn: 'glyphicon-heart'},
+	 {stateOff: 'glyphicon-off'}
+ ];
+
+ function getSimilar(){
```

Comment:
@johnhillphoto Would it make sense to put this getSimilar function into the state as a resolve? That way, it will be available to the controller as soon as it loads and you can simply attach the result to the $scope.similarExperiences?